ansible/roles/developer_packages/tasks/main.yml

```diff
@@ -43,3 +43,5 @@
       - python
       - python-pip
       - locate
+      - curl
```

Comment:
weird how this appears to be installed in 16.04 build, yet needs manual adding here